static/sass/shared-css.js

```diff
@@ -0,0 +1,127 @@
+import { css } from "lit";
+import {VARS} from "./_vars-css.js";
+
+export const SHARED_STYLES = [ 
+  VARS,
+  css`
+
+  * {
+    box-sizing: border-box;
+    list-style: none;
+    font: inherit;
+    text-decoration: inherit;
+    -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+  }
+
+  .conditional-comma:last-child {
+    display: none;
+  }
+
+  h1,
+  h2,
+  h3,
+  h4 {
+    font-weight: 300;
+  }
+
+  h1 {
+    font-size: 30px;
+  }
+
+  h2,
+  h3,
+  h4 {
+    background: var(--heading-background);
+    color: var(--heading-color);
+  }
+
+  h2 {
+    font-size: 25px;
+  }
+
+  h3 {
+    font-size: 20px;
+  }
+
+  a {
+    text-decoration: none;
+    color: var(--link-color);
+  }
+  a:hover {
+    text-decoration: underline;
+    color: var(--link-hover-color);
+    cursor: pointer;
+  }
+
+  input:not([type="submit"]),
+  textarea {
+    border: 1px solid $bar-border-color;
```

Comment:
This is an example of a SCSS variable that could be replaced with a CSS variable.  We define most of our CSS variables in theme.css.